kernel/arch/dreamcast/include/dc/biosfont.h

```diff
@@ -413,14 +465,35 @@ unsigned char bfont_draw_wide(void *buffer, uint32 bufwidth, uint8 opaque,
     \param bpp              The number of bits per pixel in the buffer.
     \param opaque           If non-zero, overwrite background areas with black,
                             otherwise do not change them from what they are.
-    \param str              The string to draw.
+    \param fmt              The printf-style format string to draw.
+    \param ...              Additional printf-style variadic arguments
+
+    \sa bfont_draw_str_ex_va()
+*/
+void bfont_draw_str_ex(void *b, uint32_t width, uint32_t fg, uint32_t bg, uint8_t bpp,
+                       bool opaque, const char *fmt, ...) __printflike(7, 8);
+
+/** \brief   Draw a full formatted string to any sort of buffer (with va_args).
+
+    This function is equivalnet to bfont_draw_str_ex(), except that the variadic
+    argument list is passed via a pointer to a va_list.
+
+    \param b                The buffer to draw to.
+    \param width            The width of the buffer in pixels.
+    \param fg               The foreground color to use.
+    \param bg               The background color to use.
+    \param bpp              The number of bits per pixel in the buffer.
+    \param opaque           If non-zero, overwrite background areas with black,
+                            otherwise do not change them from what they are.
+    \param fmt              The printf-style format string to draw.
+    \param ...              Additional printf-style variadic arguments
```

Comment:
Should this be var_args ? or does ... still work?

Reply:
uhhhhhhhhhhhhh..... good question.

Reply:
Yeah, it works as intended. Just checked.